src/verify/verify.test.ts

```diff
@@ -258,7 +258,7 @@ describe("isValidVc", () => {
         fetch: mockedFetch,
       })
     ).rejects.toThrow(
-      `The VC service provider ${MOCK_VC.issuer} does not advertize for a verifier service in its .well-known/vc-configuration document`
+      `The VC service provider ${MOCK_VC.issuer} does not advertise for a verifier service in its .well-known/vc-configuration document`
```

Comment:
Should this be changed? I think we are supposed to use US vs UK english. Maybe its just a setting in the spell check settings?